### DISCUSSION TITLE
resources for Inuinnaqtun

### DISCUSSION THREAD

**Discussion by MaryCarstensen** [dotnet/ResXResourceManager#540] [Q&A]
I have enjoyed using ResXManager, and am wondering if/when the language Inuinnaqtun will be available.

Azure uses the code ikt, and has auto-translation services for it. Perhaps the code would be ikt-CA as the language Inuinnaqtun is spoken in Canada.

**Comment by tom-englert** [comment]
It supports any language that NetFramework supports

See #417
